cli/emit.rs

```diff
@@ -40,6 +40,8 @@ use std::result;
 use std::sync::Arc;
 use std::time::Instant;
 
+pub(crate) static DEFAULT_JSX_IMPORT_SOURCE: &str = "https://esm.sh/preact";
```

Comment:
Not in favour of having a default, much less a remote registry we have no control over.

Reply:
I can eliminate this. Need to figure out how to error if none is provided but there is a setting that requires it.